app/models/grda_warehouse/auth_policies/null_pii_policy.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class GrdaWarehouse::AuthPolicies::NullPiiPolicy
```

Comment:
Is this really a `false` policy?  I would think a `null` policy might have unknown consequences.

Either way, can you add a comment about what this is doing and when it is appropriate to use?

Reply:
Sure thing, added a comment, renamed to to DenyPiiPolicy to better match the behavior